.github/workflows/Shared-Release-On-RiseClipseWeb.yml

```diff
@@ -0,0 +1,77 @@
+# *************************************************************************
+# **  Copyright (c) 2022 CentraleSupélec & EDF.
+# **  All rights reserved. This program and the accompanying materials
+# **  are made available under the terms of the Eclipse Public License v2.0
+# **  which accompanies this distribution, and is available at
+# **  https://www.eclipse.org/legal/epl-v20.html
+# ** 
+# **  This file is part of the RiseClipse tool
+# **  
+# **  Contributors:
+# **      Computer Science Department, CentraleSupélec
+# **      EDF R&D
+# **  Contacts:
+# **      dominique.marcadet@centralesupelec.fr
+# **      aurelie.dehouck-neveu@edf.fr
+# **  Web site:
+# **      https://riseclipse.github.io
+# *************************************************************************
+
+name: Release on RiseClipseWeb
+
+# This CD workflow push the newly released jar to RiseClipseWeb
+
+# Secrets are not passed when calling reusable workflows, so you must pass them from caller
+on:
+  workflow_call:
+    inputs:
+      dest_folder:
+        description: Destination folder on the remote machine
+        required: true
+    secrets:
+      RISECLIPSE_WEB_HOSTNAME:
+        required: true
+      RISECLIPSE_WEB_USERNAME:
+        required: true
+      RISECLIPSE_WEB_SSHKEY:
+        required: true
+      RISECLIPSE_WEB_PASSPHRASE:
+        required: true
+      RISECLIPSE_WEB_FOLDER:
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup Environment
+      run: sudo apt-get install rename
+
+    - name: Checkout ${{ github.event.repository.name }}
+      uses: actions/checkout@v2
+      with:
+        path: ${{ github.event.repository.name }}
+
+    - name: Download artifacts
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        github_token: ${{secrets.GITHUB_TOKEN}}
+        workflow: Pre-Release-and-Prepare-Next-Dev.yml
+        workflow_conclusion: success
+        path: ${{ github.workspace }}/${{ github.event.repository.name }}/artifacts
+
+    - name: Rename file to remove release number
+      run: |
+        cd ${{ github.workspace }}/${{ github.event.repository.name }}/artifacts
+        rename 's/-\d+\.\d+\.\d+//' *.jar 
+
+    - name: Copy JAR file and Overwrite previous
+      uses: appleboy/scp-action@v1
+      with:
+        host: ${{ secrets.RISECLIPSE_WEB_HOSTNAME }}
+        username: ${{ secrets.RISECLIPSE_WEB_USERNAME }}
+        key: ${{ secrets.RISECLIPSE_WEB_SSHKEY }}
+        passphrase: ${{ secrets.RISECLIPSE_WEB_PASSPHRASE }}
+        source: "*.jar !*Application*"
```

Comment:
Where does this expression come from? I have tried it with drone-scp, which is behind appleboy/scp-action, but it does not work.

Reply:
It works if the space is replaced by a comma